perfmetrics/scripts/presubmit_test/pr_perf_test/build.sh

```diff
@@ -1,69 +1,104 @@
 #!/bin/bash
-# Running test only for when PR contains execute-perf-test label
+# Running test only for when PR contains execute-perf-test or execute-integration-tests label
+readonly EXECUTE_PERF_TEST_LABEL="execute-perf-test"
+readonly EXECUTE_INTEGRATION_TEST_LABEL="execute-integration-tests"
+readonly INTEGRATION_TEST_EXECUTION_TIME=24m
+
 curl https://api.github.com/repos/GoogleCloudPlatform/gcsfuse/pulls/$KOKORO_GITHUB_PULL_REQUEST_NUMBER >> pr.json
-perfTest=$(cat pr.json | grep "execute-perf-test")
+perfTest=$(cat pr.json | grep $EXECUTE_PERF_TEST_LABEL)
```

Comment:
Nit: `grep $EXECUTE_PERF_TEST_LABEL pr.json` is more efficient than `cat pr.json | grep $EXECUTE_PERF_TEST_LABEL` as the former creates only one process (grep), while latter does two (cat and grep) . Functionally, both are equivalent.